karma.conf.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable flowtype/require-valid-file-annotation, no-console */
+const { argv } = require('yargs')
+const rollupConfig = require('./rollup.config')
+
+// define browsers with the `--browsers` parameter
+const browsers = (argv.browsers || process.env.BROWSERS || 'Chrome').split(',')
+// run with `watch` using the `--watch` flag
+const watch = argv.watch
+
+module.exports = (config) => {
+  config.set({
+    frameworks: ['mocha'],
+    singleRun: !watch,
+    browsers,
+    autoWatch: watch,
+    customLaunchers: {
+      SLChrome: {
+        base: 'SauceLabs',
+        browserName: 'chrome',
+        platform: 'OS X 10.11',
+      },
+      SLFirefox: {
+        base: 'SauceLabs',
+        browserName: 'firefox',
+        platform: 'OS X 10.11',
+      },
+      SLEdge: {
+        base: 'SauceLabs',
+        browserName: 'microsoftedge',
+      },
+      SLSafari: {
+        base: 'SauceLabs',
+        browserName: 'safari',
+        platform: 'OS X 10.11',
+      },
+      SLInternetExplorer11: {
+        base: 'SauceLabs',
+        browserName: 'internet explorer',
+        version: '11',
+        platform: 'Windows 10',
+      },
+    },
+    preprocessors: {
+      './src/**/*.test.js': ['rollup'],
```

Comment:
To make it easier to debug the problem, you can replace this with `./src/**/base.test.js` (also on line 52). In this way you can test just a single test.